dvc/updater.py

```diff
@@ -24,11 +23,13 @@ class Updater(object):  # pragma: no cover
     TIMEOUT = 24 * 60 * 60  # every day
     TIMEOUT_GET = 10
 
-    def __init__(self, dvc_dir):
+    def __init__(self, dvc_dir, **kwargs):
```

Comment:
I would name it explicitly, this is shadowing the intent and tying up the interfaces.